cmd/lakectl/cmd/docs.go

```diff
@@ -53,6 +58,25 @@ will provide a link to download a preconfigured configuration file for you.
 ` + "`lakectl`" + ` configuration items can each be controlled by an environment variable. The variable name will have a prefix of
 *LAKECTL_*, followed by the name of the configuration, replacing every '.' with a '_'. Example: ` + "`LAKECTL_SERVER_ENDPOINT_URL`" + ` 
 controls ` + "`server.endpoint_url`" + `.
+
+## Running lakectl from Docker
+
+If you'd rather run ` + "`lakectl`" + ` from a Docker container you can do so by passing configuration elements as environment variables. 
+Here is an example: 
+
+` + "```" + `bash
+docker run --rm --pull always \
+          -e LAKECTL_CREDENTIALS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE \
+          -e LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY=xxxxx
+          -e LAKECTL_SERVER_ENDPOINT_URL=https://host.us-east-2.lakefscloud.io/ \
+          --entrypoint lakectl treeverse/lakefs \
+          repo list
+` + "```" + `
+
+_Bear in mind that if you are running lakeFS itself locally you will need to account for this in your networking configuration of 
+the Docker container. That is to say, ` + "`localhost`" + ` to a Docker container is itself, not the host machine on which it is running._
```

Comment:
I understand this comment, but I am not a fan of explaining Docker gotchas as part of our documentation.  Can we use positive phrasing -- what to do rather than what not to do?

Reply:
@johnnyaug [also caught this](https://github.com/treeverse/lakeFS/pull/5353#discussion_r1119840335) :) I've logged an issue to follow up.